AGENTS.md

```
@@ -0,0 +1,201 @@
+| Coding Standards | WordPress Coding Standards (WPCS) via `a8cteam51/team51-configs` |
+| API Backend | OpsOasis REST API (`opsoasis.wpspecialprojects.com`) |
+| Credentials | 1Password CLI (v8) |
+| MCP | php-mcp/server ^3.3 |
+
+**Required PHP extensions**: `gd`, `json`, `posix`, `readline`.
+
+---
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)